### PR TITLE
feat: adds Jada White date provider

### PR DESCRIPTION
**Pull request by Nelshulla**

### CODE REVIEW

File: lesson_05/types/types_app/src/main/java/com/codedifferently/lesson5/dataprovider/JadaWProvider.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package main.java.com.codedifferently.lesson5.dataprovider;
```
Comment (anthonydmays) [inline lesson_05/types/types_app/src/main/java/com/codedifferently/lesson5/dataprovider/JadaWProvider.java:1]: The package name is incorrect, please fix.

File: lesson_05/types/types_app/src/main/java/com/codedifferently/lesson5/dataprovider/ChukwumaIbezimProvider.java
Comment (anthonydmays) [inline lesson_05/types/types_app/src/main/java/com/codedifferently/lesson5/dataprovider/ChukwumaIbezimProvider.java:1]: Looks like your branch is out of date, make sure to sync and get the latest changes.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_05/types/types_app/src/main/java/com/codedifferently/lesson5/dataprovider/ChukwumaIbezimProvider.java
Comment (anthonydmays) [reply lesson_05/types/types_app/src/main/java/com/codedifferently/lesson5/dataprovider/ChukwumaIbezimProvider.java:1]: I still see a bunch of files here that you shouldn't be touching and your branch is still out of sync. Please make sure to revert files that aren't `JadaWProvider.java` and `jadawhite.json`.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Review by HeirJordan3 — APPROVED** [review]
LGTM

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request merged**